internal/datastore/mysql/migrations/zz_migration.0002_add_unique_datastore_id.go

```diff
@@ -0,0 +1,30 @@
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+)
+
+func createMetadataTable(driver *MySQLDriver) string {
+	return fmt.Sprintf(`CREATE TABLE %s (
+		unique_id VARCHAR(36) PRIMARY KEY);`,
+		driver.Metadata(),
+	)
+}
+
+func insertUniqueID(driver *MySQLDriver) string {
+	return fmt.Sprintf(`INSERT INTO %s (unique_id) VALUES ("%s");`,
+		driver.Metadata(),
+		uuid.NewString(),
+	)
+}
```

Comment:
We put special care into the design of mysql's datastore migrations to be data-migration free, as that is incompatible with migration tooling used in our production systems (`skeema`).

My suggestion is to follow the same strategy used for "tuple transaction zero": to insert on runtime. The PR https://github.com/authzed/spicedb/pull/539 offers a good foundation to do this, and there is a pattern that guarantees there is no race (i.e. several spicedb processes attempting to insert at the same time on startup).